content/post/frontend_talks_2017.md

```diff
@@ -0,0 +1,142 @@
+
+### Web accessibility. How to make a frontend for everyone?, _Juanjo Montiel_
+
+In the second morning talk, [Juanjo Montiel](https://twitter.com/kastwey), software engineer at Pasiona, talked about web accessibility and showed us the difficulties visually impaired people face browsing the web.
+
+By just asking us to raise our hands to answer a question Juanjo showed us one of the problems that visually impaired people had that can be solved with accessible technology, such as a mobile app for voting instead of just raising our hands.
+Like in the previous example, visually impaired people have a lot of problems browsing the web and using a computer but Juanjo explains how to make our websites accessible to make it easier for them.
+
+Visually impaired people use softwares called screen readers that read everything in the computer and tell it to the person. But that works well only if the web page is accessible!
+
+With just a few guidelines we can make our web pages accessible too:
+
+* Provide a good structure, visually impaired people lack the visual position information.
+* Make good use of title elements (`h1`, `h2`, ...), list elements, link elements, ...
+* Label all the images that provide information to the reader with the `alt` attribute. Those images that are decorative should have the `alt` attribute but it must be empty.
+* Use labels for your form controls and fieldsets to group your radio buttons.
+* Don't use divs for everything if you can, use buttons, links, selects, etc. But if you can't, use the `role` aria attribute.
+* Always provide an alternative to drag and drop.
+
+And many more that you can find out watching the full talk.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/feN8Nz85nlI" frameborder="0" allowfullscreen></iframe>
+
+### Animate the Web with Ember.js, _Jessica Jordan_
+
+The last long talk of the morning session was about how to make animations in the web using Ember.js, by [Jessica Jordan](https://twitter.com/jjordan_dev), software engineer at simplabs.
+
+First, she starts telling us about animations through the recent history until the times of Macromedia Flash and the appearance of the open standards we use today for animations on the web: HTML5, JS and CSS3.
+
+The first option for web animations is using the HTML5 `canvas` element to draw and animate. She also explains how to do that with ember components.
+This has a few advantages like not needing DOM operations, having a good performance and being a powerful way for the developer to animate things, but it's too complicated for creators and hard for accessibility.
+
+Then, as an alternative, we can use the WebAnimations API, which is still not supported in all browsers (but we can use with a polyfill). It's very similar to CSS3 keyframes, so for people already familiar with that it will be an easy experience. As with the `canvas` HTML5 element, she explains as well how to put all this into an Ember component.
```

Comment:
It will be an easy experience for people already familiar with CSS3 keyframes since they are very similar.
//too spanish/russian sentence structure